src/elements/kw-share-detail/kw-share-detail.html

```diff
@@ -1,101 +1,241 @@
-
-                        if (this.instance) {
-                            while (this.$['share-container'].firstChild) {
-                                this.$['share-container'].removeChild(this.$['share-container'].firstChild);
-                            }
-                        }
-
-                        this.instance = this.stamp({
-                            share: this.share
-                        });
-
-                        this.$['share-container'].appendChild(this.instance.root);
-                        this.detailsLoaded = true;
-                    }, () => {}, true);
+			.bottom .content {
+				@apply --layout-horizontal;
+				margin-top: 16px;
+				margin-bottom: 16px;
+			}
+			
+			.bottom .content .description {
+				@apply(--layout-flex-3);
+				color: rgba(97, 97, 97, 0.14);
+			}
+			
+			.bottom .content .description h3 {
+				margin: 0;
+				font-size: 16px;
+				line-height: 23px;
+				color: #435055;
+				font-family: "bariol";
+			}
+			
+			.bottom .content .description p {
+				color: #616161;
+				margin-right: 16px;
+				font-family: "bariol";
+				font-weight: normal;
+				font-size: 16px;
+				margin-top: 8px;
+				margin-bottom: 8px;
+			}
+			
+			.bottom .content .actions {
+				width: 25%;
+				@apply --layout-horizontal;
+				@apply(--layout-center);
+				@apply(--layout-end-justified);
+				padding-top: 5%;
+			}
+			
+			.bottom .content .actions button img {
+				position: relative;
+				top: 2px;
+				left: 2px;
+			}
+			
+			.orangebutton {
+				background-color: #FF8413;
+				border-radius: 5px;
+				width: 79px;
+				height: 32px;
+				color: white;
+				border: none;
+			}
+			
+			.remixbutton {
+				width: 79px;
+			}
+			
+			.bottom .line {
+				height: 1px;
+				width: 100%;
+				display: block;
+				background-color: rgba(99, 107, 126, .24);
+			}
+			
+			div[tombstone].bottom .likes,
+			div[tombstone].bottom .comments,
+			div[tombstone].bottom .shares {
+				width: 80px;
+				height: 20px;
+				background: rgba(0, 0, 0, 0.01);
+				color: transparent;
+			}
+			
+			.orangebutton {
+				background-color: #FF8413;
+				border-radius: 5px;
+				width: 79px;
+				height: 32px;
+				color: white;
+				border: none;
+			}
+			
+			.remixbutton {
+				width: 79px;
+			}
+			.playercontrol {
+				@apply --layout-horizontal;
+				@apply(--layout-center);
+				width: 100%;
+				margin: 9px;
+			}
+			
+			.playercontrol .pause {
+				@apply(--layout-justified);
+			}
+			
+			.playercontrol .replay {
+				@apply(--layout-justified);
+			}
+			
+			.playercontrol .lineV {
+				@apply(--layout-justified);
+				height: 20px;
+				width: 1px;
+				display: block;
+				background-color: rgba(99, 107, 126, .24);
+				margin-right: 16px;
+				margin-left: 16px;
+			}
+			
+			.playercontrol .fullscreen {
+				@apply --layout-end-justified;
+				padding-right: 9px;
+			}
+			
+			.space {
+				@apply(--layout-flex);
+			}
+		</style>
+		<kw-share-shell id$="[[share.slug]]" share="[[share]]">
```

Comment:
Design wise it might be more efficient to provide to the shell the title, date and avatar as three separate properties. They will be parsed as String and not as a bit Object